trends_presentation/src/commonMain/kotlin/net/thechance/mena/trends/presentation/screen/category_publish/CategoryPublishState.kt

```diff
@@ -0,0 +1,15 @@
+package net.thechance.mena.trends.presentation.screen.category_publish
+
+import net.thechance.mena.trends.presentation.shared.base.ErrorState
+import net.thechance.mena.trends.presentation.shared.model.CategoryUiState
+import net.thechance.mena.trends.presentation.shared.model.Selectable
+
+internal data class CategoryPublishState(
+    val isLoading: Boolean = true,
+    val error: ErrorState? = null,
+    val categories: List<Selectable<CategoryUiState>> = emptyList(),
+    val isPublishButtonLoadingVisible: Boolean = false
```

Comment:
I suggest rename this state to :
```suggestion
    val isButtonEnabled: Boolean = false 
```

Reply:
@HendSayed25  this state not for the button itself, try to control click on it and see where it is used

Reply:
okay , I understand